1a.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <vector> 
+using namespace std;
+
+int main()
+{
+	vector<int> v1(5);
+	cout <<"Размер вектора: " <<v1.size()<<"\n";
+	cout <<"Коллекция чисел: \n";
+	int i;
```

Comment:
If you don't use it outside of the for scope, there is no purpose to declare this variable in main function scope.